src/main/java/fiftyfive/administration/usermanagement/implemention/UserService.java

```diff
@@ -0,0 +1,58 @@
+package fiftyfive.administration.usermanagement.implemention;
+
+
+import fiftyfive.administration.usermanagement.constant.Constant;
+import fiftyfive.administration.usermanagement.dto.CreateUserRequest;
+import fiftyfive.administration.usermanagement.dto.UpdateUserRequestData;
+import fiftyfive.administration.usermanagement.dto.User;
+import fiftyfive.administration.usermanagement.dto.UserResponseData;
+import fiftyfive.administration.usermanagement.exception.RecordAlreadyExistsException;
+import fiftyfive.administration.usermanagement.exception.UserNotExistsException;
+import fiftyfive.administration.usermanagement.mapper.UserRequestMapper;
+import fiftyfive.administration.usermanagement.utility.UserValidation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class UserService {
+    private final UserRequestMapper userRequestMapper = new UserRequestMapper();
+
+    @Autowired
+    UserValidation userValidation;
+    Constant constant = new Constant();
+
+    public UserResponseData createUser(CreateUserRequest createUserRequest, List<User> users) throws RecordAlreadyExistsException {
+
+        for (User existingUser : users) {
+            if (existingUser.getUsername().equals(createUserRequest.getUsername())) {
+                throw new RecordAlreadyExistsException(constant.USER_ALREADY_EXISTS + createUserRequest.getUsername());
```

Comment:
```suggestion
                throw new RecordAlreadyExistsException(String.format(constant.USER_ALREADY_EXISTS, createUserRequest.getUsername()));
```

Use string formater instead of string concatenation